subcommittees/mentoring/README.md

```diff
@@ -1,73 +1,86 @@
 # The Mentoring Subcommittee
 
 ## Purpose and Scope
+
 The Mentoring Subcommittee was established in 2015 to develop and maintain a mentorship program to support instructors as they progress through training, teaching, curriculum development, and other community-related activities. We help promote community-building and networking by providing (virtual) spaces where instructors from all over the world can share teaching success stories and discuss strategies for overcoming challenges. 
 
-## What We Do 
+## What We Do
 
 ### Instructor Discussion Sessions
-Each week, we host four instructor discussion sessions. The sessions provide a forum where experienced and novice instructors can discuss past and/or upcoming workshops. Instructors who have just taught, are about to teach, or have just completed instructor training will  receive an email invitation to these meetings, but these sessions are open to anyone. 
 
-To find an upcoming session, visit this link: http://pad.software-carpentry.org/instructor-discussion
+Each week, volunteers host two instructor discussion sessions. The sessions provide a forum where experienced and novice instructors can discuss past and/or upcoming workshops. Instructors who have just taught, are about to teach, or have just completed instructor training will  receive an email invitation to these meetings, but these sessions are open to anyone. 
 
-### Instructor-Helper Retreat
-In November 2015, Software Carpentry and Data Carpentry instructors and helpers got together together at sites around the world for a day of sharing skills, trying out new lessons and ideas and discussing all things instructor and helper related. 
+To find an upcoming session, visit this link: [http://pad.software-carpentry.org/instructor-discussion][discussion]
```

Comment:
Using:

    [http://pad.software-carpentry.org/instructor-discussion][discussion]
    …
    [discussion]: http://pad.software-carpentry.org/instructor-discussion

Gives you two places to change if/when that link changes.  It's usually easier to read and maintain if you avoid mentioning the URL in the link text.  So maybe something like:

    Find upcoming sessions [here][discussion].
    …
    [discussion]: http://pad.software-carpentry.org/instructor-discussion